writing-apps/code-style.md

```diff
@@ -310,7 +310,7 @@ print ("Hello World");
 
 ## String Formatting
 
-Avoid using literals when formatting strings:
+Avoid using literals when formatting strings, as they cannot be [localized](https://wiki.gnome.org/TranslationProject/DevGuidelines):
```

Comment:
How about using "translated" instead of "localised"? That would not need any link.

Reply:
Hey there, feels like localized is more precise. The strings can be translated (if someone translates them) but they cannot be localized (ie., those translations cannot be used within the app).

Perhaps: “…as they will not appear in the list of translatable strings for localization.”

Reply:
Hmm, not sure.  I am trying to find a non-technical wording so that no explanatory link is necessary.  How about something like "... as any translations will not appear in the app"